R/setPrior.R

```diff
@@ -0,0 +1,105 @@
+
+#priors() method for stanmodule, hazardlink , longmodel and osmodel
+
+# getter method for LongModel:
+setGeneric(name = "prior", def = function(object) standardGeneric("prior"))
+#' get method for longmodel
+setMethod(
+    f = "prior",
+    signature = list(object = "LongModel"),
+    definition = function(object) {
+        object@stan@priors
```

Comment:
for all classes which are not `StanModule` I would reuse the method, i.e.
```suggestion
        priors(object@stan)
```
this simplifies maintenance - if we decide later to store the priors differently within `StanModule` then we only need to change that single method and not everywhere

Reply:
Thanks for the suggestion! I will modify accordingly.